qa/tasks/cephfs/test_mirroring.py

```diff
@@ -626,6 +652,12 @@ def test_cephfs_mirror_cancel_sync(self):
 
         snap_list = self.mount_b.ls(path='d0/.snap')
         self.assertTrue('snap0' not in snap_list)
+
+        # check sync_failures
+        res = self.mirror_daemon_command(f'counter dump for fs: {self.primary_fs_name}', 'counter', 'dump')
+        vafter = res[TestMirroring.PERF_COUNTER_KEY_NAME_CEPHFS_MIRROR_PEER][0]
+        self.assertGreater(vafter["counters"]["sync_failures"], vbefore["counters"]["sync_failures"])
```

Comment:
Wouldn't it suffice to check if the failure count has increases than validating it being greater than it was before?

Reply:
Do you mean change it to

`self.assertEquals(vafter["counters"]["sync_failures"], before["counters"]["sync_failures"] + 1)` ?

Reply:
If this is the first and only failure then > 0 would suffice IMO.

Reply:
Updated